.github/workflows/release.yaml

```diff
@@ -39,10 +23,15 @@ jobs:
         with:
           fetch-depth: 0
       - name: Select charmhub channel
-        uses: canonical/charming-actions/channel@2.1.1
+        uses: canonical/charming-actions/channel@2.2.2
         id: channel
+      - name: Release any bumped charm libs
+        uses: canonical/charming-actions/release-libraries@2.2.0
```

Comment:
This might fireback, like https://github.com/canonical/data-platform-libs/actions/runs/4067139974/jobs/7004849787

But let's give it a try. LGTM.

BTW, why not version 2.2.2?

Reply:
JFYI, 2.2.0 must not be used, see https://chat.canonical.com/canonical/pl/o7abqkmpq3y9zxmc79bwxigrdy
2.2.1+ is minimal version for us.

Resolved as you have bumbed it already.